packages/runner-ct/src/app/RunnerCt.tsx

```diff
@@ -255,8 +258,11 @@ const App: React.FC<AppProps> = observer(
           maxSize={hideIfScreenshotting(() => state.isSpecsListOpen ? 600 : 0)}
           defaultSize={hideIfScreenshotting(() => state.isSpecsListOpen ? state.specListWidth : 0)}
           onDragFinished={persistWidth('ctSpecListWidth')}
-          className="primary"
+          className={cs('primary', { 'isSpecsListClosed': !state.isSpecsListOpen })}
           // @ts-expect-error split-pane ref types are weak so we are using our custom type for ref
```

Comment:
This comment needs to be moved or removed (and it will throw a TS error right now)

Reply:
Yes, but out of scope for this PR